src/config.ts

```diff
@@ -205,6 +205,10 @@ export const PROMETHEUS_PORT: number = _.isEmpty(process.env.PROMETHEUS_PORT)
     ? 8080
     : assertEnvVarType('PROMETHEUS_PORT', process.env.PROMETHEUS_PORT, EnvVarType.Port);
 
+export const GST2_WALLET_ADDRESS: string = _.isEmpty(process.env.GST2_WALLET_ADDRESS)
```

Comment:
Same thing here about multiple networks

Reply:
Are you saying we make it required?

Reply:
just wanted to call it out, what happens if we forget to override the deployment? Any risk?

Reply:
Nothing would happen but I moved it to a chainId -> string object in constants.ts which probably makes more sense